src/asconfig.json

```diff
@@ -7,7 +7,7 @@
     "explicitStart": true,
     "exportRuntime": true,
     "initialMemory": 256,
-    "incrementalgc": true,
+    "runtime": "incremental",
```

Comment:
How about:
```js
   "gc": "none",
   "gc": "default",
   "gc": "incremental",
```

Reply:
Not sure, since "none" in this case would not only ship no GC, but also a bump allocator instead of TLSF, affecting `heap.alloc` and `heap.free`, which are unrelated to GC. Also doesn't play so well with a custom path to a runtime (here: GC), which must implement both MM and GC.